composer.json

```diff
@@ -19,7 +19,8 @@
     "drupal/coder": "~8.2.0",
     "phpspec/phpspec": "~2.0",
     "phpunit/phpunit": "~4.0",
-    "mockery/mockery": "0.9.4"
+    "mockery/mockery": "0.9.4",
+    "drush-ops/behat-drush-endpoint": "*"
```

Comment:
We could remove it; the advantage of removing it is that not all users need it.  The advantage of including it here is that the separate installation instructions do not need to be followed if the user is managing their site with Composer -- the Drush Driver will just start working as of the next `composer update`.  The separate installation instructions will need to be followed by users who are not managing their Drupal sites with Composer, though.

It's a small extension; I'd recommend keeping it in the dependencies here.  It's completely up to you, though.